.github/workflows/reusable_dax.yml

```diff
@@ -129,7 +129,7 @@ jobs:
       - name: Run DAX tests under valgrind
         run: |
           ${{github.workspace}}/test/test_valgrind.sh ${{github.workspace}} ${{env.BUILD_DIR}} memcheck "${{env.DAX_TESTS}}"
-          ${{github.workspace}}/test/test_valgrind.sh ${{github.workspace}} ${{env.BUILD_DIR}} drd "${{env.DAX_TESTS}}"
+          ${{github.workspace}}/test/test_valgrind.sh ${{github.workspace}} ${{env.BUILD_DIR}} drdshort "${{env.DAX_TESTS}}"
```

Comment:
I wouldn't run short version on dax and multinuma as these tests:
- are not executed on nightly (they require special machines)
- they last just a few-ish seconds

Reply:
Done